app/frontend/app/routes/anime/index.js

```diff
@@ -25,7 +25,12 @@ export default Ember.Route.extend({
       return libraryEntry.save();
     }), {
       progressMessage: "Saving " + anime.get('canonicalTitle') + "...",
-      successMessage: "Saved " + anime.get('canonicalTitle') + "!"
+      successMessage: function() {
+        // update the 'full-anime' relationship, since it seems to
+        // disappear into the void
+        anime.set('libraryEntry', libraryEntry);
+        return "Saved " + anime.get('canonicalTitle') + "!";
+      }
```

Comment:
Oh, looked like a top level property in the diff snippet.
Just checked the complete file, you're right.
Looks good to me then.